src/utils/bridge.js

```diff
@@ -336,13 +337,15 @@ export function awaitGBKitGlobal( timeoutMs = 3000 ) {
  * @return {Promise<{scripts: string, styles: string, allowed_block_types: string[]}>} Promise that resolves with the assets object.
  */
 export async function fetchEditorAssets() {
-	if ( window.webkit ) {
-		return await window.webkit.messageHandlers.loadFetchedEditorAssets.postMessage(
-			{ asset: 'manifest' }
-		);
+	const gbkit = getGBKit();
+
+	// iOS provides manifest directly in GBKit configuration
+	if ( window.webkit && gbkit.manifest ) {
+		return gbkit.manifest;
 	}
 
-	const { siteApiRoot, editorAssetsEndpoint, authHeader } = getGBKit();
+	// Android fallback - fetch from API
+	const { siteApiRoot, editorAssetsEndpoint, authHeader } = gbkit;
```

Comment:
I don't believe there is a downside to de-structuring all values up front. 

```suggestion
	const { manifest, siteApiRoot, editorAssetsEndpoint, authHeader } = getGBKit();

	// iOS provides manifest directly in GBKit configuration
	if ( window.webkit && manifest ) {
		return manifest;
}

	// Android fallback - fetch from API
```

Alternatively, I don't believe there is a downside to invoking `getGBKit()` multiple times, as it [early returns](https://github.com/wordpress-mobile/GutenbergKit/blob/dac7dc6beda3915fbca83dc09bfc1aee721a440d/src/utils/bridge.js#L195-L197) with the cached value on the `window`. 


```suggestion
	const { manifest } = getGBKit();

	// iOS provides manifest directly in GBKit configuration
	if ( window.webkit && gbkit.manifest ) {
		return gbkit.manifest;
}

	// Android fallback - fetch from API
	const { siteApiRoot, editorAssetsEndpoint, authHeader } = getGBKit();
```

Reply:
I reverted this change together with how manifest is passed to the editor (now it again uses separate post message scheme).

Reply:
Would you please explain the rationale for reverting for my own education? 🙇🏻‍♂️